docs/end-to-end-demo-rpi4.md

```diff
@@ -34,109 +43,147 @@ This demo will demonstrate how to get Akri working on a **Raspberry Pi 4**, all
-    More information about the Akri Helm charts can be found in the [user guide](./user-guide.md#understanding-akri-helm-charts).
 
-1. Inspect the two instances, seeing that the correct devnodes in the metadata and that one of the usage slots for each instance was reseved for this node.
-    ```sh 
-    kubectl get akrii -o yaml
+## Investigating Akri
+After installing Akri, since the /dev/video1 and /dev/video2 devices are running on this node, the Akri Agent will discover them and create an Instance for each camera. 
+
+1. List all that Akri has automatically created and deployed, namely the Akri Configuration we created when installing Akri, two Instances (which are the Akri custom resource that represents each device), two broker Pods (one for each camera), a service for each broker Pod, and a service for all brokers.
+
+    ```sh
+    watch microk8s kubectl get pods,akric,akrii,services -o wide
     ```
-1. Deploy the streaming web application and watch a pod spin up for the app.
+    For K3s and vanilla Kubernetes
     ```sh
-    # This file url is not available while the Akri repo is private.  To get a valid url, open 
-    # https://github.com/deislabs/akri/blob/main/deployment/samples/akri-video-streaming-app.yaml
-    # and click the "Raw" button ... this will generate a link with a token that can be used below.
-    curl -o akri-video-streaming-app.yaml <RAW LINK WITH TOKEN>
-    kubectl apply -f akri-video-streaming-app.yaml
-    watch kubectl get pods -o wide
+    watch kubectl get pods,akric,akrii,services -o wide
     ```
-1. Determine which port the service is running on.
+Look at the Configuration and Instances in more detail. 
+1. Inspect the Configuration that was created via the Akri udev Helm template and values that were set when installing Akri by running the following.
     ```sh
-    kubectl get services
+    kubectl get akric -o yaml
     ```
-    Something like the following will be displayed. The ids of the camera services (`udev-camera-<id>-svc`) will likely be different as they are determined by hostname.
+1. Inspect the two Instances. Notice that in the metadata of each instance, you can see the device nodes (`/dev/video1` or `/dev/video2`) that the Instance represents. This metadata of each Instance was passed to it's broker Pod as an environment variable. This told the broker which device to connect to. We can also see in the Instance a usage slot and that it was reserved for this node. If this was a shared device (such as an IP camera), you could have increased the number of nodes that could use the same device (via `--set <protocol>.capacity=2 for two nodes) and more usage slots would have been created in the Instance. Each Instance represents a device and its usage.
```

Comment:
Might want to specify udev.capacity here ... and then say that there is a capacity parameter for each protocol.

Reply:
Specify it in the install command to be the default of `--set udev.capacity=1`?

Reply:
Clarified what `capacity` is in 23fde04. How does that look?